runtime-modules/content/src/lib.rs

```diff
@@ -1255,15 +1258,52 @@ decl_module! {
         }
 
         #[weight = 10_000_000] // TODO: adjust Weight
-        pub fn update_commitment(
+        pub fn update_channel_payouts(
```

Comment:
Rust doesn't format the macro code. Please, do this manually in all places respecting the rust code length.

Reply:
Done in https://github.com/Joystream/joystream/pull/3444/commits/2b96f1d4d761e0c3cd360fa44502f1fb43026b52